components/esp_modem/examples/linux_modem/CMakeLists.txt

```diff
@@ -2,7 +2,7 @@ cmake_minimum_required(VERSION 3.5)
 
 include($ENV{IDF_PATH}/tools/cmake/project.cmake)
 
-set(EXTRA_COMPONENT_DIRS ../.. ../../port/linux)
+set(EXTRA_COMPONENT_DIRS ../../port/linux)
```

Comment:
Shall we move this to component manager file?

Reply:
Ivan addressed only target related examples in the PR and since this change is already quite big, I'd leave it as is.

We already have @suren-gabrielyan-espressif working on updating linux ports, so this might not be necessary in future (will take it from IDF)